.github/workflows/ci.yml

```
@@ -19,9 +19,15 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - uses: pnpm/action-setup@646cdf48217256a3d0b80361c5a50727664284f2
+      with:
+        version: 6.10.0
```

This seems like its outdated. The latest is 6.32.3

```suggestion
        version: 6.32.3
```

---

true! I copied pasted this and forgot to change the version